fdbserver/VersionedBTree.actor.cpp

```diff
@@ -11448,8 +11443,8 @@ TEST_CASE("/redwood/correctness/EnforceEncodingType") {
 		                               {}, // encryptionMode
 		                               reopenEncodingType,
 		                               encryptionKeyProviders.at(reopenEncodingType));
-		wait(kvs->init());
 		try {
+			wait(kvs->init());
```

Comment:
does this mean `init` can throw errors, which it wasn't doing earlier, OR, the code now does better error handling or logging?

Reply:
See my other reply. I moved the encoding type check to `init` so it will be failing in this unit test.